renderer/index.js

```diff
@@ -533,11 +534,25 @@ function closePlayer () {
   if (state.window.isFullScreen) {
     dispatch('toggleFullScreen', false)
   }
+
+  unblockPowerSave()
   restoreBounds()
   stopServer()
   update()
 }
 
+function blockPowerSave () {
+  powerSaveBlockIDs.push(powerSaveBlocker.start('prevent-display-sleep'))
+}
+
+function unblockPowerSave () {
+  powerSaveBlockIDs.filter((id) => {
+    return powerSaveBlocker.isStarted(id)
+  }).map((id) => { powerSaveBlocker.stop(id) })
+
+  powerSaveBlockIDs = []
+}
```

Comment:
This is a little weird but feels better to me than reassigning a variable every time it's stopped. If we ever have multiple video players this could expand to deal with that but I think it's sufficient for now.